.circleci/config.yml

```
@@ -0,0 +1,40 @@
+version: 2.1
+commands:
+  commoncmd:
+    steps:
+      - run: sudo apt update
+      - run: sudo apt install -qq shellcheck
+      - checkout
+      - restore_cache:
+          keys:
+            - dependency-cache-{{ checksum "package.json" }}
+      - run: npm install
+      - save_cache:
+          key: dependency-cache-{{ checksum "package.json" }}
+          paths:
+            - ./node_modules
+      - run: npm run build
+jobs:
+  check:
```

```suggestion
  lint:
```

Will make it easier to work out what's going on.